content/docs/run-core-node/installation.mdx

```diff
@@ -7,15 +7,34 @@ There are three ways to install Stellar Core: you can use a Docker image, use pr
 
 ## Docker-based installation
 
-SDF maintains a [quickstart image](https://github.com/stellar/docker-stellar-core-horizon) that bundles Stellar Core with Horizon and postgreSQL databases. It's a quick way to set up a default, non-validating, ephemeral configuration that should work for most developers.
+### Development environments
 
-The SDF also maintains a Stellar-Core-only [standalone image](https://github.com/stellar/docker-stellar-core) that starts a 3 node local stellar-core network, all on the same docker host
+SDF maintains a [quickstart image](https://github.com/stellar/docker-stellar-core-horizon) that bundles Stellar Core with Horizon and postgreSQL databases. It's a quick way to set up a default, non-validating, ephemeral configuration that should work for most developers.
```

Comment:
This is still the old Watcher model, right? i.e. Horizon + standalone Stellar-Core? I think we'd ideally we'd either (a) outline that this isn't recommended way to run Horizon or (b) talk about how to run the Core-only portion here.

Reply:
Yes it is still the same image, we're not touching it at this point. I think the diff here is confusing, I did not touch this part of the document.